CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 - Add `beforeLayerId` and `afterLayerId` options to `carto.viz.Layer.addTo` method, to customize layer position
 - Widget synchronization when using remote dataviews ([#76](https://github.com/CartoDB/web-sdk/pull/76/))
 - `icon` style helper 
+- Adding new data state in Layer ([#84](https://github.com/CartoDB/web-sdk/pull/84)) for sending data events when data is ready or updated
```

Comment:
The state of Layer is relevant, but specially for us, not so much externally for the end user. I would add a note instead more on the line of 'New data events in Layer: XXX an YYY'... what do you think?

We didn't add the PR in other notes, but it's nice, we can try to add it always.